lib/paper_trail/multi.ex

```diff
@@ -162,8 +162,8 @@ defmodule PaperTrail.Multi do
 
       _ ->
         multi
+        |> Ecto.Multi.insert_all(version_key, Version, entries_query, returning: returning)
```

Comment:
just curious: is there a reason for inserting Versions before doing the update?

Reply:
Yes, this is a test for the case: https://github.com/jungsoft/paper_trail/pull/2/files#diff-37c9c2e2780268ed3baadb82751de315360e4ce88d02bb6d085e435c694fbd64R677


Basically it's because when the queryable given to the `update_all` use a field in the where clause that is modified by the `set` in the update. So if we have a query for users with `where is_active == false` and then `set: [is_activ: true]`, the result of the query will change after the update, so the `insert_all` of the versions will not function properly. 

Do you see a problem with this approach? Another approach would be first query for the ids of the changed records